proxy/src/main/java/com/velocitypowered/proxy/command/VelocityCommandManager.java

```diff
@@ -211,17 +209,12 @@ public boolean executeImmediately(final CommandSource source, final String cmdLi
     Preconditions.checkNotNull(source, "source");
     Preconditions.checkNotNull(cmdLine, "cmdLine");
 
-    ParseResults<CommandSource> parse = parse(cmdLine, source, false);
+    ParseResults<CommandSource> parse =
```

Comment:
The command line is intentionally not normalized for tab completion (spaces matter).